ibazel/ibazel.go

```diff
@@ -339,26 +353,31 @@ func verb(s string) string {
 }
 
 func (i *IBazel) build(targets ...string) (*bytes.Buffer, error) {
-	b := i.newBazel()
+	if i.bazel != nil {
```

Comment:
This kills the `i.bazel` indiscriminately, should there be a check to make sure short circuiting is enabled?

Reply:
done, thanks for catching that!